schema/codelists/interestType.csv

```diff
@@ -4,11 +4,10 @@ votingRights,Voting rights,"A controlling interest in an entity gained by holdin
 appointmentOfBoard,Appointment of board,A controlling interest in an entity. Defined as the absolute right to appoint members of the board of directors.,
 otherInfluenceOrControl,Other influence or control,"Any influence or control in an entity that is distinct from being a shareholder, having voting rights or having the absolute right to appoint to the board.",
 seniorManagingOfficial,Senior managing official,A controlling interest in an entity gained by employment. Defined as the person who exercises control over the management of the entity.,
-settlorOfTrust,Settlor of trust,"A controlling interest in a trust. Defined as the person who gives property into trust for the benefit of beneficiaries. In some legal systems, a settlor is also referred to as a trustor, or occasionally, a grantor or donor.",
-trusteeOfTrust,Trustee of a trust.,A controlling interest in a trust. Defined as a person or firm that holds and administers property or assets for the benefit of a third party.,
-protectorOfTrust,Protector of a trust,A controlling interest in a trust. Defined as a person appointed under the trust instrument to direct or restrain the trustees in relation to their administration of the trust.,
-beneficiaryOfTrust,Beneficiary of a trust,An economic interest in a trust. Defined as a person or legal entity who profits from a trust's holdings or activities.,
-otherInfluenceOrControlOfTrust,Other influence or control of a trust,"Any influence or control in a trust that is distinct from being a settlor, trustee, protector or beneficiary.",
+settlor,Settlor,"A settlor is a person who, either actually or by operation of law, creates a trust. A settlor of a trust is sometimes also known by other names such as a trustor, a grantor or a donor. Also to be used for a person with a settlor-equivalent role in any legal arrangement that is similar to a trust.",
+trustee,Trustee,A trustee is a person who administers the trust for the benefit of a third party and in whom the legal title of the trust property is vested either by declaration of the settlor or by operation of law. Also to be used for a person with a trustee-equivalent role in any legal arrangement that is similar to a trust.,
+protector,Protector,"A protector is a person appointed to protect the interests or wishes of the settlor, providing influence and guidance to the trustee who administers the trust. Also to be used for a person with a protector-equivalent role in any legal arrangement that is similar to a trust.",
+beneficiaryOfLegalArrangement,Beneficiary of a legal arrangement,An economic interest in a trust or other legal arrangement. Defined as a person or legal entity who profits from a trust or other legal arrangement's holdings or activities.,
```

Comment:
Tiny question: would it be better to use 'benefits' rather than 'profits' here. The latter is more strongly tied to only financial gain, whereas the former is more inclusive of usage rights etc.